client/post-editor/controller.js

```diff
@@ -91,13 +92,21 @@ function maybeRedirect( context, postType, site ) {
 	return false;
 }
 
-function getPressThisContent( text, url, title ) {
-	return ReactDomServer.renderToStaticMarkup(
-		<p>
-			{ text ? <blockquote>{ text }</blockquote> : null }
-			via <a href={ url }>{ title }</a>.
-		</p>
-	);
+function getPressThisContent( text, url, title, image, embed ) {
+	let pieces = [];
```

Comment:
I really wouldn't use `const` to try to indicate immutability. That just leads people down the wrong road w.r.t. what it really does.